components/auth/ImpersonateAuth.php

```diff
@@ -43,7 +43,7 @@ public function authenticate($user, $request, $response)
 
             if ($accessToken && ($identity = $accessToken->user)) {
                 $user->login($identity);
-                Yii::$app->user->isImpersonatedViaRest = true;
+                Yii::$app->user->isImpersonated = true;
```

Comment:
@gevorgmansuryan Since this attribute will only be available with core version `v1.16.0`+, we have to increase the `minVersion` in the module in the file `module.json`. Can you please do that as well?